tests/testthat/test-summary-log.R

```diff
@@ -77,6 +77,28 @@ withr::with_options(list(rbabylon.bbi_exe_path = read_bbi_path(),
     expect_identical(sum_df$yaml_md5, ALL_MODS_YAML_MD5)
   })
 
+  test_that("summary_log() parses heuristics correctly", {
```

Comment:
If we are adding new columns to the output of `add_summary()`, then we should have tests covering that specifically (nothing prevents the implementation from being different than for `summary_log()`).

Reply:
I'm not sure I totally follow you. We don't check all the columns as is. If we want to test the full structure we should probably use a golden file of some sort. I'm ok with that, but the downside is that it makes the tests more brittle (for example, all of our golden file tests were broken when a tibble update changed the way they formatted row names or something like that). I could go either way honestly, but my preference is to _not_ go golden file and take the hit on safety.

If I'm missing your point please clarify. Thanks.

Reply:
ahhhh... I missed that you were talking about `add_summary()`. 

Uh, yea that's fine if you think we should do these same tests on `add_summary()` too. Honestly it feels unnecessary to me though, given that `add_summary()` [claims to return](https://github.com/metrumresearchgroup/rbabylon/blob/develop/R/summary-log.R#L31) everything `summary_log()` returns, joined onto the input tibble. We should probably explicitly test that claim, but we do at least test that it has the [right number of columns](https://github.com/metrumresearchgroup/rbabylon/blob/develop/tests/testthat/test-summary-log.R#L73) so that's not nothing. 

I would support adding that explicit test though, to be sure that `add_summary()` has all the columns in `summary_log()` plus whatever its input was. That feels more reasonable than repeating these exact tests for `add_summary()`